public_suffix_list.dat

```diff
@@ -698,6 +698,7 @@ of.by
 // bz : https://www.iana.org/domains/root/db/bz.html
 // http://www.belizenic.bz/
 bz
+co.bz
```

Comment:
The selected line `+co.bz` adds a new public suffix entry for the `co.bz` domain under the Belize (`bz`) top-level domain. This pull request started with IANA and then the registrry as defined by the IANA db entry for .bz, which then has the name in the dropdown and their policy pages at the authoritative registry.

change indicates that `co.bz` is now recognized as a valid public suffix within the public suffix list.